docs/guide/web-simulation.md

```diff
@@ -324,6 +324,68 @@ This folder also contains a `server.sh` utility script to automatically start an
 
 Please make sure that the `WEBOTS_HOME` variable is set before running the simulation and session server scripts.
 
+#### Using Docker
+
+The simulation server can also be run in a Docker container.
+In this section we provide a sample Docker setup that could be used to run the simulation on localhost.
+
+You can use the following `Dockerfile` to build your Docker image.
+```
+FROM cyberbotics/webots:R2021a-ubuntu18.04
+
+RUN apt update
+RUN apt install -y firejail python3-pip
+RUN pip3 install tornado pynvml psutil requests distro
+ENV AUDIODEV=null
+ENV DEBIAN_FRONTEND=noninteractive
+ENV DISPLAY=:99
+ENV WEBOTS_HOME=/usr/local/webots
+ENV QTWEBENGINE_DISABLE_SANDBOX=1
```

Comment:
Isn't this variable inherited from the parent container image?

Reply:
Probably yes. I will check and remove all the unneeded variables redefinition.